include/cuco/detail/static_map/static_map_ref.inl

```diff
@@ -668,6 +764,8 @@ class operator_impl<
     auto probing_iter      = probing_scheme(group, key, storage_ref.window_extent());
     auto const empty_value = ref_.impl_.empty_slot_sentinel().second;
 
+    auto constexpr use_insert = PayloadWrite and (sizeof(value_type) > 8);
```

Comment:
What does `use_insert` mean?

Reply:
`use_insert` will actually `wait_for_payload`, as it uses an **atomic_store/insert** operation to store first element into the map, rather than an `apply`.